tests/test_runtime_http.cpp

```
@@ -1446,6 +1446,75 @@ TEST(HttpSSRF, PublicAddrInfoSynthetic) {
 
+// --- Additional SSRF range tests (IPv4) ---
+
+TEST(HttpSSRF, PrivateHostCarrierGradeNAT) {
+    EXPECT_TRUE(__ry_is_private_host("100.64.0.1", 80));
+}
+
+TEST(HttpSSRF, PrivateHostCarrierGradeNATUpper) {
+    EXPECT_TRUE(__ry_is_private_host("100.127.255.255", 80));
+}
+
+TEST(HttpSSRF, PrivateHostBenchmarking) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)